src/provider_references.jl

```diff
@@ -0,0 +1,74 @@
+import Tokenize.Tokens
```

Comment:
I guess this shouldn't really be depending on internal representations of CSTParser